specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -343,9 +343,9 @@ Programmatic security refers to security
 decisions made by security aware applications. Programmatic security is
 useful when declarative security alone is not sufficient to express the
 security model of the application. The API for programmatic security
-consists of methods of the Java EE Security API _SecurityContext_
+consists of methods of the Jakarta Security _SecurityContext_
 interface, and methods of the EJB _EJBContext_ interface and the servlet
```

Comment:
Use of EJB here should be changed to Jakarta Enterprise Beans as it is referring to the Jakarta Enterprise Beans api rather than the JCP api

Reply:
When you changed Java EE for Jakarta EE you missed the "EE". Isn't it supposed to be "Jakarta EE Security..."?

Reply:
No, it's just "Jakarta Security".

Using "EJB" is fine, as long as the disclaimer is added to the beginning of the spec.